net-misc/connman/connman-1.33-r1.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+inherit autotools systemd
+
+DESCRIPTION="Provides a daemon for managing internet connections"
+HOMEPAGE="https://01.org/connman"
+SRC_URI="mirror://kernel/linux/network/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86"
+IUSE="bluetooth debug doc examples +ethernet l2tp ofono openvpn openconnect pptp policykit tools vpnc +wifi wispr"
+
+RDEPEND=">=dev-libs/glib-2.16
+	>=sys-apps/dbus-1.2.24
+	>=net-firewall/iptables-1.4.8
+	bluetooth? ( net-wireless/bluez )
+	l2tp? ( net-dialup/xl2tpd )
+	ofono? ( net-misc/ofono )
+	openconnect? ( net-misc/openconnect )
+	openvpn? ( net-misc/openvpn )
+	policykit? ( sys-auth/polkit )
+	pptp? ( net-dialup/pptpclient )
+	vpnc? ( net-misc/vpnc )
+	wifi? ( >=net-wireless/wpa_supplicant-2.0[dbus] )
+	wispr? ( net-libs/gnutls )"
+
+DEPEND="${RDEPEND}
+	>=sys-kernel/linux-headers-2.6.39
+	virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-1.32-execinfo-assumptions.patch"
+	"${FILESDIR}/${PN}-1.31-xtables.patch"
+)
+
+src_prepare() {
+	local mf="Makefile.am"
+
+	# Fix for bug # 591596
+	awk '/tmpfilesdir = @SYSTEMD_TMPFILESDIR@/{getline;next}{print}' \
+		$mf > $mf.tmp || die "Failed to remove systemd tmpfilesdir blob"
+	mv $mf.tmp $mf || die "failed to rename $mf.tmp to $mf"
+	sed -i -e 's/ProtectSystem=full/ProtectSystem=true/' \
+		src/connman.service.in || die "failed to update connman.service.in"
+
+	# Fix for bug # 596276
+	sed -i -e '/actiondir/s/polkit/polkit-gobject-1/' configure.ac ||
+		die "failed to patch polkit detection in configure.ac"
+
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--localstatedir=/var \
+		--with-systemdunitdir=$(systemd_get_systemunitdir) \
+		--with-tmpfilesdir=$(get_libdir)/tmpfiles.d \
```

Comment:
What does this do? I don't recall autotools having any sane behavior for passing relative paths. Not to mention: didn't you just comment that directory out?